Gemfile

```diff
@@ -38,7 +38,7 @@ gem 'webpacker', '~> 3.5.5'
 # http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#responders
 gem 'responders', '~> 2.0'
 
-gem 'nokogiri', '~> 1.8.2'
+gem 'nokogiri', '~> 1.8.4'
```

Comment:
You might want to tweak this to be `~> 1.8.5` - there was a new security vulnerability announced in 1.8.4 a couple of weeks ago.

Reply:
sure thanks!